config/host/generic/client_host_packages.repos

```diff
@@ -13,7 +13,7 @@ repositories:
   uros/rclc:
     type: git
     url: https://github.com/ros2/rclc.git
-    version: foxy
+    version: master
```

Comment:
@pablogs9 Did you intend to change the branch direclty to  `rolling` (aka master branch) or do you rather want to use the `galactic` branch?

Reply:
It is correct, for  this repo `main` branch we use rclc `master`, for `galactic` branch we use rclc `galactic`